Fix .transform() argument

tritemio opened the pull request:
In the last section of `lesson3-head-pose.ipynb` about data augmentation, the `.transform()` method was not getting the variable `tfms` but the default `get_transform()`.

With the correction, I get a warning (about not be able to create a data batch) that I left in the notebook in case is important.

jph00 commented:
Thanks for this. There's a lot of extra metadata changes in this diff. Could you please provide a PR that only includes the actual code changes you made? You may find this helpful: https://docs.fast.ai/dev/develop.html

tritemio commented:
@jph00 I stripped the metadata.

In the mods I added a paragraph and enabled the custom data transform. I still left the warning (I don't know if it is totally benign) and replaced the last output image because changing the data augmentation changes the effect on the pictures.

jph00 commented:
There's still quite a bit of extra stuff in your diff unrelated to this change. Having said that, we don't really want something that creates that warning, so instead I'll just remove the unused 'tfms' cell. Thanks a lot for bringing this issue to our attention!
